prac_08/convert_miles_km.py

```diff
@@ -0,0 +1,36 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.core.window import Window
+
+FACTOR_MILES_TO_KM = 1.60934
+
+
+class MilesConverterApp(App):
+    """Convert miles to kilometres"""
+
```

Comment:
This is where you put the output_km string property:
```
output_km = StringProperty()
```
Now the "output_label" in the kv file will show the output_km string and automatically update when it changes.